osu.Game.Rulesets.Osu/Objects/Drawables/DrawableRepeatPoint.cs

```diff
@@ -35,15 +38,29 @@ public DrawableRepeatPoint(RepeatPoint repeatPoint, DrawableSlider drawableSlide
             Blending = BlendingParameters.Additive;
             Origin = Anchor.Centre;
 
-            InternalChild = scaleContainer = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.ReverseArrow), _ => new SpriteIcon
+            InternalChild = scaleContainer = new Container
             {
                 RelativeSizeAxes = Axes.Both,
-                Icon = FontAwesome.Solid.ChevronRight,
-                Size = new Vector2(0.35f)
-            }, confineMode: ConfineMode.NoScaling)
-            {
                 Anchor = Anchor.Centre,
                 Origin = Anchor.Centre,
+                Children = new Drawable[]
+                {
+                    tail = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.SliderTail), _ => null)
```

Comment:
I think this should be included inside the legacy implementation of `ReverseArrow` instead. You should be able to DI in the parenting hitobject and get the rotation from there.

Eventually I think we'll rename `ReverseArrow` to `RepeatPoint` or something like that.